tests/std/tests/P0896R4_ranges_alg_move/test.cpp

```diff
@@ -18,66 +20,63 @@ struct int_wrapper {
         val = std::exchange(that.val, -1);
         return *this;
     }
+    auto operator<=>(const int_wrapper&) const = default;

```

Comment:
`const int_wrapper` is inconsistently `const`-west when the rest of this file is east `const`. Please pick one or the other. 

FWIW I'd prefer for all tests to be consistently `const`-west - yes, despite that it's my fault this test is east-`const` - but I won't require you to fix that pre-existing issue.

Reply:
Other reviewers: Yes, any use of `<=>` requires `<compare>` to be included. WG21 made a last-minute change for C++20 to have a slew of headers include `<compare>` transitively in LWG-3330. `<ranges>` and `<utility>` are two of those headers, so this is fine.